app/presenters/provider/admin/account/users_index_presenter.rb

```diff
@@ -7,8 +7,10 @@ def initialize(current_user:, users:, params:)
     @ability = Ability.new(current_user)
 
     pagination_params = { page: params[:page] || 1, per_page: params[:per_page] || 20 }
+    sorting_params = "id asc"
 
-    @users = users.paginate(pagination_params)
+    @users = users.order(sorting_params)
+                  .paginate(pagination_params)
```

Comment:
I'm usually against adding order to queries just to fix tests unless it is an improvement to UX. Sorting by id I don't think is really an improvment, or is it? 

I think we had some cucumber steps that compared table content without regards to order. Would that be possible here?

Reply:
Yeah, we could solve this in the tests, but think about the user experience...

You have a list of elements, you reload the page, and the order is suddenly different :upside_down_face: 

Especially if you have multiple pages - if you have an unconsistent order, then you might have very unexpected results when scrolling through the pages.
I mean... it's unlikely - in the same DB engine it is likely you will always get the same order. But that's not guaranteed!
So, I think a predictable ordering is not only good for tests, but for user experience.

Reply:
Sorting by ID is roughly sorting by "from oldest to newest", as the ID is incrementing. I explained in the PR description that `created_at` sometimes is not reliable.

Reply:
To me it makes sense to use name or something for sorting. Also perhaps you want to see last entries first. idk.

Reply:
There was an attempt to make the columns sortable, but it was a bit too much effort for a feature that nobody asked about.
For this one I prefer order in which the users were created. As for asc/desc I indeeed doubted which one to pick, but for this specific page "first created - first in the list" somehow seemed to be more meaningful to me :shrug: We can discuss again and change if needed.